app/enterprise/changelog.md

```diff
@@ -129,9 +129,10 @@ for details.
     parameter. Now you can provide status code ranges and single status codes
     together (e.g., `201-204,401`)
   - **Request Validator** (_NEW_):
-    - Validate the request body against a JSON schema; if the payload is not
+    - Validate the request body against a schema; if the payload is not
     valid JSON or if it doesn't conform to the schema, the plugin returns a 400
```

Comment:
shouldn't we remove `is not valid JSON` as well?

Reply:
This PR only clarifies that the schema format used is not JSON schema (http://json-schema.org); but the request body still is validated, both syntactically (valid JSON) and semantically (conforms to the schema spec), so we shoudn't remove that.

Reply:
Understood, thank you for clarifying.